doc/model.md

```diff
@@ -0,0 +1,74 @@
+# Istio Service Model
+
+Istio's services model introduces the concept of a service version, a
```

Comment:
Could just say "Istio introduces the concept of a _service version_ whc is a fine-grained way to subdivide